Makefile

```diff
@@ -57,13 +57,9 @@ ISTANBUL_FLAGS := \
 # Tasks
 ##
 
-# Install node modules.
-node_modules: package.json $(wildcard node_modules/*/package.json)
-	@npm install
-	@touch $@
-
 # Install dependencies.
-install: node_modules
+install:
```

Comment:
I think developers are likely to want to run `yarn install/make install` locally, so I think `make install` should just run `yarn install`, but let's update the CI task to use `yarn install --frozen-lockfile`. 

What do you think?

Reply:
Makes sense, fixed!